app/uk/gov/hmrc/gform/sharedmodel/formtemplate/destinations/Destination.scala

```diff
@@ -27,16 +27,26 @@ import cats.data.NonEmptyList
 import JsonUtils.nelFormat
 import uk.gov.hmrc.gform.sharedmodel.email.LocalisedEmailTemplateId
 import uk.gov.hmrc.gform.sharedmodel.form.FormStatus
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.destinations.DestinationIncludeIf.{ IncludeIfValue, StringValue }
 import uk.gov.hmrc.gform.sharedmodel.formtemplate.destinations.Destination.SubmissionConsolidator
 import uk.gov.hmrc.gform.sharedmodel.notifier.NotifierPersonalisationFieldId
 
 sealed trait DestinationWithCustomerId {
   def customerId(): Expr
 }
 
+sealed trait DestinationIncludeIf extends Product with Serializable
+
+object DestinationIncludeIf {
+  case class StringValue(value: String) extends DestinationIncludeIf
```

Comment:
If I'm not mistaken wrapper value is turned to boolean by handlebars in the end, so I wonder if it would it be possible to name this `HandlebarValue` or something like this instead of `StringValue`?